spec/models/lti_client_spec.rb

```diff
@@ -0,0 +1,37 @@
+describe LtiClient do
+  describe 'uniqueness_validation' do
+    subject { create :lti_client }
+    it { is_expected.to validate_uniqueness_of(:client_id).scoped_to(:host) }
+  end
+  describe '#get_oauth_token' do
+    # let!(:lti_client) { create :lti_client }
```

Comment:
I'm not sure if this comment means the approach didn't work. It seems to me that we can use `let` here (`@lti_client` below doesn't seem to be anything special), which would be preferred